pkg/webhook/work/mutating.go

```diff
@@ -3,6 +3,8 @@ package work
 import (
 	"context"
 	"encoding/json"
+	"github.com/karmada-io/karmada/pkg/util"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Please group the imports. [here is an example](https://github.com/karmada-io/karmada/pull/526#discussion_r669591653).